content/blog/validating-smi-conformance-with-meshery.md

```diff
@@ -0,0 +1,70 @@
+
+- Versioned suite of conformance tests.
+- Self-reporting with guaranteed provenance and integrity of conformance test results.
+- Continually published conformance results.
+- Support for all service meshes that implement SMI.
+
+Delivering on these requirements required a tool specifically suited to SMI.
+
+![Meshery](/img/blog/validating-smi-conformance-with-meshery/meshery-logo-light-text-side.svg#center)
+
+## SMI Conformance Tool: Meshery
+
+Meshery, the service mesh management plane, is service mesh agnostic, which is an important characteristic as a compliance tool. No better tool for the job (in fact, no other tool capable of the job) than Meshery, which has been [a partner of SMI](https://meshery.io/blog/a-standard-interface-for-service-meshes) since its inception.
+
+![SMI Conformance Tool](/img/blog/validating-smi-conformance-with-meshery/meshery-service-mesh-compliance-tool.png).
+
+Meshery is ideal tooling in that it provides lifecycle management of a large number of service meshes and sample applications which need to be provisioned, tested, and deprovisioned in the process of validating conformance. And that Meshery is capable of generating load and verifying test assertions. Meshery will be extended to validate any service mesh that claims to implement and conform SMI specifications. And, more broadly, to leverage the same tooling to validate service mesh standards at-large, including SMI and [Service Mesh Performance](https://smp-spec.io) (SMP).
+
+## Defining Conformance
+
+Conformance of SMI specifications is defined as a series of test assertions. A test assertion is a condition that must be positively verified in order for an implementation to be considered conformant. A test assertion may involve any number of conditions. Sets of test assertions are categorized by SMI specification. Collectively, these test sets comprise the complete suite of SMI conformance tests. Just like SMI itself, the suite of [SMI Conformance tests](https://layer5.io/projects/service-mesh-interface-conformance) is versioned, and with each new version of SMI, as interfaces added and specifications changed, the Conformance requirements will change as appropriate. The SMI community is the change controller and overseers of what it means to conform to SMI. Work on the mechanics of the conformance tests occurs in the [CNCF Service Mesh WG](https://github.com/cncf/sig-network/tree/master/service-mesh-wg) develops the process and policy around the conformance program.
```

Comment:
> as interfaces added and specifications changed,

as interfaces **are** added and specifications changed,

> overseers of what it means

**oversees what** it means

> [CNCF Service Mesh WG](https://github.com/cncf/sig-network/tree/master/service-mesh-wg) develops

[CNCF Service Mesh WG](https://github.com/cncf/sig-network/tree/master/service-mesh-wg) **which** develops

Reply:
Thank you, thank you.